rest-api/api/participant_summary_api.py

```diff
@@ -1,16 +1,56 @@
-from api.base_api import BaseApi
-from api_util import PTC_AND_HEALTHPRO
-from app_util import auth_required
+from api.base_api import BaseApi, make_sync_results_for_request
+from api_util import PTC_HEALTHPRO_AWARDEE, AWARDEE, DEV_MAIL
+from app_util import auth_required, get_validated_user_info
 from dao.participant_summary_dao import ParticipantSummaryDao
+from flask import request
+from werkzeug.exceptions import Forbidden, InternalServerError
 
 
 class ParticipantSummaryApi(BaseApi):
   def __init__(self):
     super(ParticipantSummaryApi, self).__init__(ParticipantSummaryDao())
 
-  @auth_required(PTC_AND_HEALTHPRO)
+  @auth_required(PTC_HEALTHPRO_AWARDEE)
```

Comment:
This logic is getting complicated enough that at some point we probably will want some unit tests of it, or API tests that use a user other than DEV_MAIL.... we should have tests for the other paths.

Reply:
agreed, I'll make a ticket.